distributed/loadbalancer.md

```diff
@@ -0,0 +1,197 @@
+# The Gravwell Load Balancer
+
+To make setting up your environment as easy as possible, Gravwell provides a custom load balancer specifically designed for use with Gravwell webservers. It can automatically discover Gravwell webservers, meaning you don't need to reconfigure the loadbalancer every time you add or remove a webserver--and if a webserver goes down, it will automatically direct users to another server.
+
+## Load Balancer Architecture
+
+The load balancer is an HTTP(S) proxy which automatically directs clients to one of the Gravwell webservers. It sets a cookie on the user's browser to maintain a level of "stickiness", so one session's requests all go to the same webserver.
+
+The load balancer discovers Gravwell webservers by communicating with the Gravwell  (see [the distributed webserver documentation](frontend.md) for more info about the datastore).
```

Comment:
with the Gravwell.... what?

Reply:
Forgot to finish the sentence. Stand by. I'll also fix the typo you introduced a minute ago ;)